Week-5/Work & HW/index/blackjack.html

```diff
@@ -0,0 +1,245 @@
+              for (let y = 0; y < array.length; y++) {
+                sum += array[y];
+                parseInt(sum);
+                if (sum <= 10) {
+                  array[x] = 11
+                } else {
+                  array[x] = 1;
+                }
+                check21Bot(sum);
+              }
+              array2 = sum;
+          } else if (array[x] == "KING") {
+              array[x] = 10;
+              let sum = 0;
+              for (let y = 0; y < array.length; y++) {
+                sum += array[y];
+                parseInt(sum);
+              }
+              check21Bot(sum);
+          } else if (array[x] == "QUEEN") {
+              array[x] = 10;
+              let sum = 0;
+              for (let y = 0; y < array.length; y++) {
+                sum += array[y];
+                parseInt(sum);
+              }
+              check21Bot(sum);
+          } else if (array[x] == "JACK") {
+              array[x] = 10;
+              let sum = 0;
+              for (let y = 0; y < array.length; y++) {
+                sum += array[y];
+                parseInt(sum);
+              }
+              check21Bot(sum);
+          } else {
+            let sum = 0;
+            for (let y = 0; y < array.length; y++) {
+              sum += array[y];
+              parseInt(sum);
+            }
+            check21Bot(sum);
+          }
+        }
+      }
+
+      $.get("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1", (response) => {
+        $.get(`https://deckofcardsapi.com/api/deck/${response.deck_id}/draw/?count=2`, (response2) => {
+          let card1image = response2.cards[0].image;
+          let card2image = response2.cards[1].image;
+          let card1 = response2.cards[0].value;
+          let card2 = response2.cards[1].value;
+          checkNum(card1, cards);
+          checkNum(card2, cards);
+          check(cards)
+          cardsimages.push(card1image, card2image);
+          $("#div").append(`<img src = '${card1image}'/>
+                            <img src = '${card2image}'/>`);
+          $.get(`https://deckofcardsapi.com/api/deck/${response.deck_id}/draw/?count=2`, (response3) => {
+            let card1image = response3.cards[0].image;
+            let card2image = response3.cards[1].image;
+            let card1 = response3.cards[0].value;
+            let card2 = response3.cards[1].value;
+            cards2images.push(card1image, card2image);
+            checkNum(card1, cards2.cards);
+            checkNum(card2, cards2.cards);
+            checkBot(cards2.cards)
+            $("#div2").append(`<img id = 'hidden' src = 'http://ic.pics.livejournal.com/dailyafirmation/691132/529371/529371_original.jpg' height = 300px/>
+                               <img src = '${card2image}' height = 300px/>`);
+          })
+          $("#hit").click(() => {
+            $.get(`https://deckofcardsapi.com/api/deck/${response.deck_id}/draw/?count=1`, (response2) => {
```

Comment:
I think you already have a response2 in this context. Instead you should probably just take the entire `$("#hit").click(() => { .... }` and put it in the top level of your script. It doesn't need to be inside the response function.